java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -39,12 +39,31 @@
  * and call incRefCount to increment the reference count.
  */
 public final class HostColumnVector extends HostColumnVectorCore {
+  /**
+   * Interface to handle events for this HostColumnVector. Only invoked during
+   * close, hence `onClosed` is the only event.
+   */
+  public interface EventHandler {
+    /**
+     * `onClosed` is invoked with the updated `refCount` during `close`.
+     * The last invocation of `onClosed` will be with `refCount=0`.
+     *
+     * @note the callback is invoked with this `HostColumnVector`'s lock held.
```

Comment:
```suggestion
     * @note The callback is invoked with this `HostColumnVector`'s lock held.
```

Reply:
We tend to favor starting with lowercase and we also favor indenting the description so that it reads as one block, but I do think the hyphen is discouraged. How about a compromise? (https://github.com/rapidsai/cudf/pull/13898/commits/a0944eebf501006f9f542e09a9dfc1934dbdf4b2)

Reply:
It's not very important since we are not using the generated docds. But if we are the users who read the generated docs, we will see they are ugly with bad style: Sentences start without first letter being Capped.

Reply:
and sentences starting without cap. letter is not good IMO.

Reply:
Since our code styles on this are all over the place, I am going to suggest filing a follow on if you feel strongly about it. We should then in that issue fix all the comments and make sure that the pre commit hooks exercise them, so we have something consistent and easy to test.